content/en/docs/concepts/workloads/pods/sidecar-containers.md

```diff
@@ -0,0 +1,73 @@
+---
+title: API for sidecar containers
```

Comment:
In the `concepts/workloads/pods` section, it is weird to focus on the API of a resource rather than the resource itself.

We may want to simplify to "Sidecar containers".

Reply:
@sftim,
Can you please share your thoughts on this.

Reply:
Yes: @tengqm was right.

Reply:
Thanks, Reverted back to "Sidecar Containers".